picamera2/outputs/fileoutput.py

```diff
@@ -31,14 +33,20 @@ def fileoutput(self):
     @fileoutput.setter
     def fileoutput(self, file):
```

Comment:
Maybe there's a small enhancement we could make here.

If we accept a `split` parameter (defaulting to `None`) then the caller could explicitly supply an integer which would be the size we split packets into. Below, where we detect the UDP socket we could change the `None` into 65507.

The benefit would be that users can choose a size that is better for their own network, for example they might know that there's an MTU of 1500 that gets enforced somewhere, and so this might behave better for them.

Reply:
Yep, that's a good point, I'll add a split=None, to the FileOutput constructor.

I assume we only need to make use of the split value for UDP packets, as there wouldn't 
be any advantage for TCP etc. sockets?

Reply:
Yes, I think that's right, TCP will sort itself out.